### PR TITLE
463480/feat  audit user agent

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/messaging/send-audit.js [hunk 1]
```diff
@@ -326,6 +326,33 @@ const sendViewToAudit = async (pk, type, subject, details, { username, displayna
   await sendEvent(event)
 }
 
+const sendLoginToAudit = async ({ username, displayname }, userAgent) => {
```
Comment (jbarnsley10) [inline app/messaging/send-audit.js:329]: There is a method sendViewToAudit() line 305 in this file already. Can we re-use that call with appropriate params

File: app/routes/user.js [hunk 2]
```diff
@@ -15,12 +15,15 @@ const { sendEmail } = require('../messaging/send-email')
...
 const { sendReportSomethingEmails, createAuditsForReportSomething } = require('../lib/email-helper')
+const { lookupPoliceForceByEmail } = require('../repos/police-forces')
+const { sendLoginToAudit } = require('../messaging/send-audit')
 
 module.exports = [
   {
     method: 'POST',
     path: '/user',
     options: {
+      auth: { scope: [scopes.admin] },
```
Comment (jbarnsley10) [inline app/routes/user.js:26]: I thought these had been added in a previous PR? Did I miss something?

File: app/routes/user.js [hunk 3]
```diff
@@ -144,6 +146,8 @@ module.exports = [
       }
     },
     handler: async (request, h) => {
+      await sendLoginToAudit(getCallingUser(request), request.headers['enforcement-user-agent'])
```
Comment (jbarnsley10) [inline app/routes/user.js:149]: I think this call is used on nearly every GET in enforcement, or at least more regularly than jkust the first login of a session

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
I think you need to trigger the audit at a different point in the process

File: app/routes/user.js (hunk 2 above)
Comment (whitewaterdesign) [reply app/routes/user.js:24]: moved them to somewhere more visible

**Review by jbarnsley10 — APPROVED** [review]
That's great now

**Pull request merged**